dotCMS/src/integration-test/java/com/dotmarketing/filters/FiltersTest.java

```diff
@@ -694,7 +657,7 @@ public void shouldRedirect401() throws IOException {
      * /products/
      */
     @Test
-    public void shouldRedirectToFolderIndex() throws IOException {
+    public void shouldRedirectToFolderIndex() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)